src/main/java/org/mycore/mets/model/header/MetsHdr.java

```diff
@@ -19,16 +22,25 @@
  */
 public class MetsHdr implements IMetsElement {
 
+    public static final DateTimeFormatter DT_FORMATTER = new DateTimeFormatterBuilder()
+        .appendPattern("uuuu-MM-dd")
+        .appendLiteral('T')
+        .appendPattern("HH:mm:ss")
+        .toFormatter();
```

Comment:
CREATEDATE and LASTMODDATE are of type xsd:dateTime this Formatter is not sufficient